recipes/r-ltmle/meta.yaml

```
@@ -0,0 +1,76 @@
+{% set version = '1.3-0' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-ltmle
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/ltmle_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/ltmle/ltmle_{{ version }}.tar.gz
+  sha256: fb31d0dd6ab81687b81f3279b414c21e91c655e10aac12f73fc6723efd848aad
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  noarch: generic
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-matrix
+    - r-matrixstats
+  run:
+    - r-base
+    - r-matrix
+    - r-matrixstats
+
+test:
+  commands:
+    - $R -e "library('ltmle')"           # [not win]
+    - "\"%R%\" -e \"library('ltmle')\""  # [win]
+
+about:
+  home: https://github.com/joshuaschwab/ltmle, http://joshuaschwab.github.io/ltmle/
```

Let's split the URLs and use HTTPS

```suggestion
home: https://github.com/joshuaschwab/ltmle
doc_url: https://joshuaschwab.github.io/ltmle/
```